src/installer/managed/Microsoft.NET.HostModel/MachO/CodeDirectoryHeader.cs

```diff
@@ -0,0 +1,121 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.NET.HostModel.MachO;
+
+/// <summary>
+/// Version 0x20400
+/// </summary>
+[StructLayout(LayoutKind.Sequential)]
+internal struct CodeDirectoryHeader
+{
+    private BlobMagic _magic;
```

Comment:
Reading https://github.com/apple-oss-distributions/Security/blob/3dab46a11f45f2ffdbd70e2127cc5a8ce4a1f222/SecurityTool/sharedTool/codesign.c#L73
it doesn't look like this can be anything other than CSMAGIC_CODEDIRECTORY. Is there a reason to allow people to set anything else?

Reply:
No real reason, I can change it to readonly with a default value.